src/Augmentation/AugmentationHelpers.jsx

```diff
@@ -281,8 +281,8 @@ function initAugmentations() {
 
     const NanofiberWeave = new Augmentation({
         name:AugmentationNames.NanofiberWeave, repCost:3.75e4, moneyCost:1.25e8,
-        info:"Synthetic nanofibers are woven into the skin's extracellular matrix using electrospinning. " +
-             "This improves the skin's ability to regenerate itself and protect the body from external stresses and forces.<br><br>" +
+        info:"Synthetic nanofibers are woven into the skin's extracellular matrix using electrospinning, " +
+             "which improves regeneration and extracellular homeostasis.<br><br>" +
```

Comment:
I'd rephrase to something like

"Synthetic nanofibers are woven into the skin's extracellular matrix using electrospinning, which improves its regenerative and extracellular homeostasis abilities"